react-redux/contactsMVC/contacts.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import RaisedButton from 'material-ui/RaisedButton';
+import injectTapEventPlugin from 'react-tap-event-plugin'
+
+import Conversation from './components/Conversation';
+
+const CommentList = (state = [], action) => {
+  switch(action.type) {
+    case 'ADD_COMMENT':
+      return [
+        ...state,
+        {
+          id: action.id,
+          username: action.username,
+          thumbnail: action.thumbnail,
+          comment: action.comment
+        }
+      ];
+    case 'DEL_COMMENT':
+      return state.filter( s => s.id !== action.id );
+    default:
+      return state;
+  }
+}
```

Comment:
put your reducer into another file. you may call it reducer.js in small project.
in large project, you may organise your project in the way introduced here
https://github.com/erikras/ducks-modular-redux